From 29b7738085c4fb272b845900ef908f621d02cd68 Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Wed, 09 May 2012 09:16:33 +0000
Subject: fdo#46509: xsltml .xsl files missing from wiki-publisher.oxt

A regression introduced by e7a1e17c4fdea4103e7f921e8cd9c315c1dbe612 "convert
swext to gbuild and add to tail_build."

Change-Id: I29efe28ad6353d725afe9cce6f470c5492aa87a8
---
diff --git a/swext/Extension_wiki-publisher.mk b/swext/Extension_wiki-publisher.mk
index 1b6eb51..f142446 100644
--- a/swext/Extension_wiki-publisher.mk
+++ b/swext/Extension_wiki-publisher.mk
@@ -53,6 +53,13 @@ $(eval $(call gb_Extension_add_file,wiki-publisher,WikiEditor/Module1.xba,$(WIKI
 $(eval $(call gb_Extension_add_file,wiki-publisher,WikiEditor/script.xlb,$(WIKISRC)/dialogs/script.xlb))
 $(eval $(call gb_Extension_add_file,wiki-publisher,WikiEditor/dialog.xlb,$(WIKISRC)/dialogs/dialog.xlb))
 $(eval $(call gb_Extension_add_file,wiki-publisher,WikiEditor/SendToMediaWiki.xdl,$(WIKISRC)/dialogs/SendToMediaWiki.xdl))
+$(eval $(call gb_Extension_add_file,wiki-publisher,filter/math/cmarkup.xsl,$(OUTDIR)/bin/xslt/export/xsltml/cmarkup.xsl))
+$(eval $(call gb_Extension_add_file,wiki-publisher,filter/math/entities.xsl,$(OUTDIR)/bin/xslt/export/xsltml/entities.xsl))
+$(eval $(call gb_Extension_add_file,wiki-publisher,filter/math/glayout.xsl,$(OUTDIR)/bin/xslt/export/xsltml/glayout.xsl))
+$(eval $(call gb_Extension_add_file,wiki-publisher,filter/math/mmltex.xsl,$(OUTDIR)/bin/xslt/export/xsltml/mmltex.xsl))
+$(eval $(call gb_Extension_add_file,wiki-publisher,filter/math/scripts.xsl,$(OUTDIR)/bin/xslt/export/xsltml/scripts.xsl))
+$(eval $(call gb_Extension_add_file,wiki-publisher,filter/math/tables.xsl,$(OUTDIR)/bin/xslt/export/xsltml/tables.xsl))
+$(eval $(call gb_Extension_add_file,wiki-publisher,filter/math/tokens.xsl,$(OUTDIR)/bin/xslt/export/xsltml/tokens.xsl))
 $(eval $(call gb_Extension_add_file,wiki-publisher,filter/odt2mediawiki.xsl,$(WIKISRC)/src/filter/odt2mediawiki.xsl))
 $(eval $(call gb_Extension_add_file,wiki-publisher,help/en-US/com.sun.wiki-publisher/wikisend.xhp,$(WIKISRC)/help/wikisend.xhp))
 $(eval $(call gb_Extension_add_file,wiki-publisher,help/en-US/com.sun.wiki-publisher/wikiformats.xhp,$(WIKISRC)/help/wikiformats.xhp))
--
cgit v0.9.0.2-2-gbebe
